src/main/resources/application.properties

```diff
@@ -1 +1,24 @@
+server.port= 8085
+## Stripe API
+stripe.key.publishable=pk_test_ZPLk0o0siEhURnWP3O31PAxl003mdYeZGh
+stripe.key.secret=sk_test_68AFqEufTbTSfjsBBmPC8hdd00xKfiq7ze
```

Comment:
i guess this should be private info(use local variables)

Reply:
@edgar-kr I guess you meant environment variables

Reply:
yeah)